Marlin/ultralcd.cpp

```diff
@@ -1072,7 +1070,7 @@ static void lcd_prepare_menu() {
   // Level Bed
   //
   #if ENABLED(AUTO_BED_LEVELING_FEATURE)
-    if (axis_known_position[X_AXIS] && axis_known_position[Y_AXIS])
+    if (axis_homed[X_AXIS] && axis_homed[Y_AXIS])
       MENU_ITEM(gcode, MSG_LEVEL_BED, PSTR("G29"));
```

Comment:
True. We can drop that.

I did a more comprehensive patch-up with #3579, but I will bring in any changes from here that were missed.